app-text/mupdf/mupdf-1.21.1.ebuild

```diff
@@ -55,6 +55,10 @@ PATCHES=(
 )
 
 src_prepare() {
+	if ! use drm; then
+		PATCHES+=( "${FILESDIR}"/$P-no-drm.patch )
+	fi
+
 	default
```

Comment:
For consistency `default` should always be the first line in `src_prepare()`.

Reply:
Thanks, I moved `default` to the first line and now use `eapply` instead of `PATCHES+=...` (because `default` applies all patches in `PATCHES` and therefore runs before the conditional patch would be added to `PATCHES`).

Reply:
I'm not sure that is compatible with consistent user expectations of "all ebuild patches applied first, then my own". Anyway, I think a better fix will be to apply the patch unconditionally and have it use an ifdef - then we pass conditionally -Dfoo.

Reply:
Thanks, that's a good idea. Implemented it.